Sources/unused-imports/main.swift

```diff
@@ -51,6 +54,16 @@ private struct Configuration: Decodable {
         } else {
             self.ignoredModuleRegex = nil
         }
+
+        if let string = try values.decodeIfPresent(String.self, forKey: .reporter) {
+            if string == "json" {
+                self.reporter = JSONReporter()
+            } else {
+                self.reporter = nil
+            }
+        } else {
+            self.reporter = nil
```

Comment:
```suggestion
            self.reporter = SedCommandReporter()
```

seems fine to default this here so you can eliminate the optionality

Reply:
Absolutely.  100%.

Reply:
Updated.